CompilerSource/ipc/Room.fbs

```diff
@@ -0,0 +1,113 @@
+
+table RoomInstance {
+  id:int;
+
+  x:int;
+  y:int;
+  object_id:int;
+
+  flip_x:bool;
+  flip_y:bool;
+  scale_x:double;
+  scale_y:double;
+  rotation:double;
+  color:int;
+
+  creation_code:string;
+  locked:bool;
+}
+
+struct RoomTile {
+  id:int;
+
+  background_id:int;
+  bg_x:int;
+  bg_y:int;
+  room_x:int;
+  room_y:int;
+  width:int;
+  height:int;
+  depth:int;
+
+  scale_x:double;
+  scale_y:double;
+  color:int;

```

Comment:
For some reason they didn't give tiles rotation like they did instances. Maybe we should change that? Is it any less efficient when batching the tiles? I mean, I guess it really doesn't make sense to rotate tiles but.... it could be useful.

It seems Tiled has tile rotation:
http://discourse.mapeditor.org/t/can-i-rotate-tiles/703

So I don't see why not?